routemaster/models/client_token.rb

```diff
@@ -0,0 +1,31 @@
+require 'routemaster/models'
+require 'routemaster/mixins/redis'
+require 'securerandom'
+
+module Routemaster
+  module Models
+    class ClientToken
+      include Mixins::Redis
+
+      REDIS_KEY = 'api_keys'.freeze
+
+      def self.get_all
+        _redis.hgetall REDIS_KEY
+      end
+
+      def self.exists?(key)
+        _redis.hexists REDIS_KEY, key
+      end
+
+      def self.generate_api_key(service_name)
+        new_key = SecureRandom.uuid 
```

Comment:
I think my comment may have got lost in a different change, but this should really be `.hex(16)` rather than `.uuid`. UUIDs are effectively guaranteed to be unique, but are not guaranteed to be unguessable. 2^128 bits of random data is both unique (with at least as strong a guarantee as a v4 UUID) and unguessable, so it a better choice for a security token or API key.